<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CHNIOPEXL6113" failures="1" tests="1" name="com.cio.testcases.GoogleSearch" time="0.000" errors="0" timestamp="4 Oct 2024 06:07:43 GMT">
  <testcase classname="com.cio.testcases.GoogleSearch" name="openGoogle" time="0.000">
    <failure message="
DataProvider should be static: public java.lang.String[][] com.cio.utilities.TestUtils.getData(java.lang.reflect.Method) throws org.apache.poi.EncryptedDocumentException,java.io.IOException" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
DataProvider should be static: public java.lang.String[][] com.cio.utilities.TestUtils.getData(java.lang.reflect.Method) throws org.apache.poi.EncryptedDocumentException,java.io.IOException
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:315)
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:251)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:401)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1358)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1058)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1160)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:754)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:292)
	at org.testng.SuiteRunner.run(SuiteRunner.java:241)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1169)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1094)
	at org.testng.TestNG.run(TestNG.java:1006)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase>
</testsuite>
